### ISSUE TITLE
Setting AuthTokenEnvVar has no effect

### ISSUE THREAD

**Issue by je-al** [signalfx/gateway#195]
I'm using a docker image that derivates from quay.io/signalfx/gateway:v1.0.12, 
with the following environment (this under ECS): 
```
"Env": [
                "AWS_EXECUTION_ENV=AWS_ECS_EC2",
                "SFX_ACCESS_TOKEN=yh44k..............",
```
and configuration:
```
{
  "MaxGracefulWaitTime": "30s",
    "GracefulCheckInterval": "1s",
    "SilentGracefulTime": "3s",
    "StatsDelay": "1s",
    "ForwardTo": [
      {
        "Type": "signalfx",
        "Name": "signalfxforwarder",
        "AuthTokenEnvVar": "SFX_ACCESS_TOKEN",
        "BufferSize": 1000000,
        "DrainingThreads": 50,
        "MaxDrainSize": 5000
      }
    ],
    "ListenFrom": [
      {
        "ListenAddr": "0.0.0.0:18080",
        "Type": "signalfx",
        "AdditionalDimentions": { }
      }
    ],
    "LogDir": "-"
}
```
however, the logs show:
```
time=2019-02-26T20:38:42Z caller=ratelimit.go:36 protocol=signalfx direction=forwarder name=signalfxforwarder struct=BufferedForwarder draining_index=14 err="invalid status code 401: " message="error sending datapoints"
time=2019-02-26T20:38:43Z caller=ratelimit.go:36 protocol=signalfx direction=forwarder name=signalfxforwarder err="invalid status code 401: " message="Unable to process datapoints"
```

I've verified the access token and it is active and working.

**Comment by je-al** [comment]
I've changed my setup enough that I'm not confident I can reproduce ti, and neither I have the bandwidth to attempt to do so, so I'm closing this.

**Comment by mpetazzoni** [comment]
@je-al `AuthTokenEnvVar` is only available in 1.0.13 and above.
